ci/jenkins/jobs/macros.yaml

```diff
@@ -144,6 +144,24 @@
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
           ./ci/jenkins/test.sh --testcase windows-e2e --registry ${DOCKER_REGISTRY} --proxyall
 
+- builder:
+      name: builder-e2e-win-containerd
+      builders:
+          - shell: |-
+                #!/bin/bash
+                set -ex
+                DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+                ./ci/jenkins/test.sh --testcase windows-containerd-e2e --registry ${{DOCKER_REGISTRY}} --win-image-node '{win_image_node}'
```

Comment:
You mentioned '{win_image_node}', but why use antrea_win_image_node_name in projects-lab.yaml?

Reply:
I was trying to differentiate them, because `win_image_node` is filled with `antrea_win_image_node_name` in projects-labe.yaml, and `antrea_win_image_node_name` is defined in defaults.